src/main/java/ua/ek/pages/xpath/AppleTabletsPage.java

```diff
@@ -16,6 +16,107 @@
     @FindBy(xpath=".//div[@class='page-title']/h1[@class='t2' and contains(text(), 'Apple')]")
     private WebElement pageTitle;
 
+
+//  Страница "Планшеты Apple"
+//   https://ek.ua/list/30/apple/
+
+/*
+
+1. contains
+// Выбор всех описаний для камер для всех планшетов на странице
+
+.//div[contains(@title,'Камера')]
+.//div[contains(@class, 'm-s-f2')]/div[3]
+.//div[contains(@class, 'm-s-f2')]/child::div[3]
+.//div[contains(@class, 'm-s-f2')]/div[contains(@title,'Камера')]
+.//table[@class='model-short-block']//div[@class='model-short-description']/div[contains(@class, 'm-s-f2')]/div[3]
+
+// Выбор описания для камеры пятого по номеру планшета на странице
+".//form[@id='list_form1']/div[5]//div[contains(@title, 'Камера')]")
+
+2. starts-with
+// Название планшетов
+.//span[starts-with(text(),'Apple')]
+
+3. last
+// Выбор последнего элемента в описании
+.//div[contains(@class, 'm-s-f2')]/div[last()]
+
+4. not
+// Выбор элементов span, атрибут class которых не содержит слово 'mainmenu'
+.//span[not (contains(@class, 'mainmenu'))]
+
+5. or
+// Элемент с title, который содержит слово 'Корпус' в описании планшета
+.//div[contains(@title,'Камера35') or contains(@title,'Корпус')]
+
+6. and
+// Выбор элементов с названием планшета
+.//div[@class='page-title']/h1[@class='t2' and contains(text(), 'Apple')]
+
+7. ancestor
+// Все предки div элемента с title, который содержит слово 'Корпус' в описании
+.//div[contains(@title,'Корпус')]/ancestor::div
+
+8. parent
+// Третий по счету стикер в описании планшета
+.//div[contains(@class, 'm-s-f2')]/parent::div/div/a/following-sibling::a[2]
```

Comment:
/parent::div/div/a/following-sibling::a[2] Change this structure with using relative path and without [2]